core/rest/rest_api.json

```diff
@@ -5,7 +5,7 @@
         "description": "Interact with the enterprise blockchain through Hyperledger Fabric API",
         "version": "1.0.0"
     },
-    "host": "127.0.0.1:3000",
+    "host": "127.0.0.1:5000",
```

Comment:
Swagger gives you a UI in the browser you can use to test out an API. Because it was assumed that you would run Swagger outside of Vagrant and that you would be testing against a peer running in Vagrant, the default port was set to 3000. Now that it's 5000 everywhere, it should work no matter what your setup is, unless you've customized the REST port. @angrbrd is the Swagger expert so she can double check that I'm not making stuff up :)